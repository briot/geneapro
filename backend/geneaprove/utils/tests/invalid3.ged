0 HEAD
1 SOUR GRAMPS
1 DATE 10 NOV 2008
0 TRLR
