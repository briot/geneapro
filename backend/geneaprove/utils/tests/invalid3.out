<dir>/invalid3.ged:1 Missing 1 occurrence of GEDC in HEAD
